src/bp-core/bp-core-filters.php

```diff
@@ -780,6 +780,29 @@ function bp_setup_nav_menu_item( $menu_item ) {
 			} else {
 				$menu_item->classes = array( 'current_page_item', 'current-menu-item' );
 			}
+		} else {
+			$domain_arr = array( bp_loggedin_user_domain() );
+			if ( in_array( $current, $domain_arr ) ) {
```

Comment:
@krunal-buddyboss No need seprate variable for this.

```suggestion
			if ( in_array( $current, array( bp_loggedin_user_domain() ) ) ) {
```